packages/flutter/lib/src/widgets/transitions.dart

```diff
@@ -388,6 +389,43 @@ class RelativePositionedTransition extends AnimatedWidget {
   }
 }
 
+/// Animated version of a [DecoratedBox] that animates the different properties
+/// of its [Decoration].
+class DecoratedBoxTransition extends AnimatedWidget {
+  /// Creates an animated [DecorationBox] whose [Decoration] animation updates
+  /// the widget. 
+  /// 
+  /// `decoration` and `position` cannot be null.
```

Comment:
The backtick alone will make the text in "code" font, which is usually what you want for arguments.  In this case, there just happens to be a relevant thing to link to with the same name, so we cheat slightly and link to the property when documenting the argument.  (Usability over theoretical purity)